e2e/tests/app.e2e.js

```diff
@@ -0,0 +1,43 @@
+import { test, expect } from '@playwright/test';
```

Comment:
We can, there is a [sort-imports](https://eslint.org/docs/latest/rules/sort-imports) eslint rule that will enable reporting of such errors. The only question is how we want to configure it, because the possibilities are many.

Please check [angularjs-angular-hybrid-angular-cli eslint configuration](https://github.com/xlts-dev/angularjs-angular-hybrid-angular-cli/pull/2/files#diff-6884918dc8291219be508e05e28965b958c734def85324f3b53858ea4702090fR120)

Reply:
Any config is better than nothing. I have preferences, but the important thing is having determinism and consistency.